apps/gitness/src/pages-v2/pull-request/hooks/usePRChecksDecision.ts

```diff
@@ -0,0 +1,181 @@
+import { useEffect, useMemo, useState } from 'react'
+
+import pluralize from 'pluralize'
+
+import { TypesPullReq, TypesRepository, useChecksPullReqQuery } from '@harnessio/code-service-client'
+import { ExecutionState } from '@harnessio/views'
+
+import { determineStatusMessage, generateStatusSummary } from '../../../pages/pull-request/utils'
+
+interface Check {
+  check: {
+    status: ExecutionState
+  }
+}
+
+interface Data {
+  checks: Check[]
+}
+export function usePRChecksDecision({
+  repoMetadata,
+  pullReqMetadata
+}: {
+  repoMetadata: TypesRepository | undefined
+  pullReqMetadata: TypesPullReq | undefined
+}) {
+  const {
+    data: { body: data } = {},
+    error,
+    refetch
+  } = useChecksPullReqQuery({
+    repo_ref: `${repoMetadata?.path}/+`,
+    pullreq_number: pullReqMetadata?.number ?? 0,
+    queryParams: { debounce: 500 }
+  })
+  const [count, setCount] = useState(DEFAULT_COUNTS)
+  const [color, setColor] = useState<string>('GREEN_500')
+  const [background, setBackground] = useState<string>('GREEN_50')
```

Comment:
I don't think either of these colours exist 🤔

Reply:
done